<launch>
  <arg name="SIMULATE_ROBOT" value="true"/>

  <!-- spacenav -->
  <node pkg="spacenav_node" type="spacenav_node" name="spacenav_node" output="screen"/>
  <node pkg="jsk_teleop_joy" type="spacenav2pose.py" name="spacenav2pose" output="screen">
    <param name="pose" value="/jsk_interactive_marker_manipulation/move_marker"/>
  </node>
  

  <!-- interactive marker -->
  <include file="$(find jsk_interactive_marker)/launch/interactive_marker_controller_remote.launch">
    <arg name="REMOTE_TF" value="/tf"/>
  </include>

  <group if="$(arg SIMULATE_ROBOT)">
    <include file="$(find jsk_interactive_marker)/launch/pr2_eus_tf_publisher.launch"/>

    <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher" args="_source_list:='[joints_set]'">
      <param name="use_gui" type="boolean" value="true"/>
    </node>

    <node pkg="roseus" type="roseus" args="$(find jsk_interactive)/euslisp/move-joint-interface-robot.l" name="move_joint_interface_robot">
      <remap from="/joint_states" to="/joints_set"/>
    </node>

  </group>

</launch>